### ISSUE TITLE
There is a problem with the code

### ISSUE THREAD

**Issue by genbei** [TharinduDR/TransQuest#23]
I have a problem when I look at the code, because the paper mentioned that sentence-level prediction in monotranquest model uses [cls], but I see this in the code.`TransQuest\transquest\algo\sentence_level\monotransquest\models\roberta_model.py`

>     
        sequence_output = outputs[0]
        logits = self.classifier(sequence_output)
        outputs = (logits,) + outputs[2:]

this is get_sequence_out()?

Please answer, thank you very much

**Comment by TharinduDR** [comment]
outputs[0] is the CLS token and self.classifier(sequence_output) add a classification layer on top of the CLS token. 
Please have a look at HuggingFace implementation - https://huggingface.co/transformers/_modules/transformers/models/roberta/modeling_roberta.html#RobertaForSequenceClassification.forward

**Comment by genbei** [comment]
> outputs[0] is the CLS token and self.classifier(sequence_output) add a classification layer on top of the CLS token.
> Please have a look at HuggingFace implementation - https://huggingface.co/transformers/_modules/transformers/models/roberta/modeling_roberta.html#RobertaForSequenceClassification.forward

but outputs[0].size()=[8,80,1024]
[batch_size, seq_length, hidden_size]

**Comment by TharinduDR** [comment]
Yeah, 1024 is the vector size of the large model. What is your problem precisely?

**Comment by genbei** [comment]
> Yeah, 1024 is the vector size of the large model. What is your problem precisely?

outputs[0] stands for word vector not Sentence vector？, because I want to make some improvements and need to understand this problem

**Comment by genbei** [comment]
> outputs[0] is the CLS token and self.classifier(sequence_output) add a classification layer on top of the CLS token.
> Please have a look at HuggingFace implementation - https://huggingface.co/transformers/_modules/transformers/models/roberta/modeling_roberta.html#RobertaForSequenceClassification.forward


I think，this is the CLS used, right
> 
pooled_output = outputs[1]

        pooled_output = self.dropout(pooled_output)
        logits = self.classifier(pooled_output)
        reshaped_logits = logits.view(-1, num_choices)

**Comment by TharinduDR** [comment]
I used the HuggingFace implementation. Please have a look at their documentation before taking decisions to change the code. https://huggingface.co/transformers/_modules/transformers/models/roberta/modeling_roberta.html#RobertaForSequenceClassification.forward. 
According to this outputs[0] is the classification token (CLS token) in RoBERTa. 
The code you have showed is how you take CLS token in BERT.

**Comment by genbei** [comment]
> I used the HuggingFace implementation. Please have a look at their documentation before taking decisions to change the code. https://huggingface.co/transformers/_modules/transformers/models/roberta/modeling_roberta.html#RobertaForSequenceClassification.forward.
> According to this outputs[0] is the classification token (CLS token) in RoBERTa.
> The code you have showed is how you take CLS token in BERT.

yes，But according to size,outputs[0] stands for word vector not Sentence vector，right？
